packages/sdk/src/actions/setRecipientForFund.ts

```diff
@@ -0,0 +1,32 @@
+import { type Address, parseAbi, type Hex, decodeFunctionData } from "viem";
+import { MANAGEMENT_FEE } from "../../tests/constants.js";
```

Comment:
You can't import from `tests` in `src` and you shouldn't import and use `MANAGEMENT_FEE` here anyways ;-)